capstone-frontend/src/App.test.js

```diff
@@ -2,8 +2,15 @@ import React from 'react';
 import { render } from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
-  const { getByText } = render(<App />);
-  const linkElement = getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+var container = document.createElement("div");
```

Comment:
nit: Avoid `var`. If this doesn't need to be reassigned later, use `const` (https://google.github.io/styleguide/jsguide.html#features-use-const-and-let).

Reply:
Got it, I thought append child would create problems with re-assignment but it doesn't. Thanks!